scopes/envs/envs/environment.ts

```diff
@@ -22,7 +39,87 @@ export interface Environment {
   [key: string]: any; // :TODO need to define an abstract type for service handlers (now using any)
 
   /**
-   * Returns the linter to use (ESLint, ...)
+   * @deprecated Fatal: a breaking API has introduced. Use getBuildPipe() instead
+   */
+  getPipe?: () => BuildTask[];
+
+  /**
+   * Returns the list of dependencies
+   * Required for any task
+   */
+  getDependencies?: () => PkgDependencies | Promise<PkgDependencies>;
```

Comment:
extract to separate interface it's not part of the env's core

Reply:
Are you sure? it seems to be used by most tasks

Reply:
yes I'm sure:) we can discuss it if you want (both this and the getPackageJsonProps)